### DISCUSSION TITLE
How can I get a sponsor Badge?

### DISCUSSION THREAD

**Discussion by MdHRShohel** [community/community#31081] [Other Feature Feedback, Questions, & Ideas] [answered]
Help to get a Sponsor Badge!

**Comment by melvincwng** [comment, marked as answer]
@MdHRShohel, you need to sponsor someone to get the badge.

Example: https://github.com/sponsors/melvincwng

Hope this clarifies your question.

**Reply by MdHRShohel** [reply]
Thank you,
But how can I post as you for a sponsor?

**Reply by melvincwng** [reply]
@MdHRShohel , glad to have helped.

For a full list of countries that are available under the GitHub Sponsors program, please refer to this link here: https://github.com/sponsors

From your profile, I see that you are from Bangladesh. Unfortunately, at this time in writing, Bangladesh doesn't seem to be one of the countries available under the GitHub Sponsors program.

Also, take note, in order to get the badge that you've mentioned earlier, you need to sponsor someone instead.

Hope this clarifies.

**Reply by MdHRShohel** [reply]
Yes. I got it.
Thanks again! Take love!
